docs/unified-test-documentation/dasharo-compatibility/31U-heads-bootloader-support.md

```diff
@@ -35,3 +35,245 @@ bootloader.
 **Expected result**
 
 The `Heads` bootloader screen should be displayed.
+
+## HDS003.001 Boot from USB option is available and works correctly
```

Comment:
In the documentation of this test case, we should add, that this option is displayed if there are no other booting options (i. e. NVME with bootable OS).

Reply:
Added